modules/cloudsql-instance/README.md

```diff
@@ -46,9 +52,15 @@ module "db" {
 
 ```hcl
 module "db" {
-  source           = "./fabric/modules/cloudsql-instance"
-  project_id       = var.project_id
-  network          = var.vpc.self_link
+  source     = "./fabric/modules/cloudsql-instance"
```

Comment:
Last thing we miss here, is an example with PSC instead of PSA. Please add if you have bandwidth, or we can add this as a follow-up.

Reply:
Done.